scikits/odes/sundials/cvode.pyx

```diff
@@ -864,6 +887,8 @@ cdef class CVODE:
 
             self.options['interruptfn'] = fn
 
+            self._old_api = False
+
```

Comment:
It probably should not go so silently, if the user expects old API. Probably some warning would be kind. Even more, we should probably issue a warning about old API being deprecated....